src/main/java/com/example/bookshop/validation/Isbn.java

```diff
@@ -0,0 +1,18 @@
+package com.example.bookshop.validation;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.hibernate.validator.internal.constraintvalidators.hv.ISBNValidator;
+
+@Constraint(validatedBy = ISBNValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Isbn {
```

Comment:
Let's use default org.hibernate.validator.constraints.ISBN annotation for validation.